app/assets/stylesheets/custom.scss

```diff
@@ -176,7 +183,16 @@ section.range-slider {
     }
   }
   
+ .button-slider-1{
+   margin-right: 100px;
+ } 
+
+ .button-slider-2{
```

Comment:
Please check the indentation

Reply:
Avoid using names like `button-slider-1`. Also, add a space before `{`